pool/app/api_key/api_key.ml

```diff
@@ -0,0 +1,8 @@
+include Entity
+include Entity_guard
+include Event
+module Actor = Actor
+
+let find = Repo.find
+let find_by_token = Repo.find_by_token
+let all = Repo.all
```

Comment:
We used these statements in several other modules, but it would be much easier just including the repo:

```suggestion
include Repo
```